yash-datalake/azuredeploy.parameters.json

```diff
@@ -2,35 +2,29 @@
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",

```

Comment:
set defaultValues for the parameters that do not need to be generated and remove them.  Use GEN values for the rest